wdl/CallSV.wdl

```diff
@@ -0,0 +1,133 @@
+version 1.0
+
+import "Structs.wdl"
+
+task PBSV {
+
+    input {
+
+        File bam
+        File bai
+
+        File ref_fasta
+        File ref_fai
+        File tandem_repeat_bed
+
+        RuntimeAttr? runtime_attr_override
+    }
+
+    Int disk_size = ceil(size(bam, "GB")) + 10
+
+    String out_prefix = basename(bam)
+
+    command <<<
+        set -euxo pipefail
+
+        num_core=$(cat /proc/cpuinfo | awk '/^processor/{print $3}' | wc -l)
```

Comment:
Ahh, this is interesting.  I'm curious - if we're setting the number of cores below in the runtime section, why do we need to detect the number of cores in the command itself?  Is this so we can change the runtime section without changing the code for the command section, and thus call caching will continue to work even if we change the number of CPUs between invocations?

Reply:
Exactly.
The default runtime (or VM specification) can be overriden by users.
But the actual command in the workflow will be OK with how many ever (but still reasonable) CPUs available to the container.

One trivial issue though, is that the PBSV docker image on quay.io is built with Busybox (embeded Linux, very lean), so other mechanisms for deciding CPU numbers are unavailable....